doc/Language/5to6-perlvar.pod6

```diff
@@ -48,21 +44,17 @@ you can call methods on it. Unlike C<$_> you cannot assume C<@_> as the
 default variable for those methods to operate on (i. e. C<@_.shift> works,
 C<.shift> does not).
 
-=head3 $LIST_SEPARATOR, $"
+=head3 X<$LIST_SEPARATOR, $"|Foreign,$" (Perl)>
 
 Currently, there is no equivalent of the List Separator variable in Raku.
 Design document L<S28|https://design.raku.org/S28.html> says there isn't one,
 so you probably don't want to hold your breath.
 
-=head3 $PROCESS_ID, $PID, $$
-
-X<|$$ (Perl)>
+=head3 X<$PROCESS_ID, $PID, $$|Foreign,$$ (Perl)>
```

Comment:
Again, three terms to index here.

Reply:
Agreed, they were not indexed before, just the one you're indexing. And badly, at that (below the title instead of above). But as long as you're re-indexing it, let's go all the way.